File: challenges/codeeval/easy/096/Skhorn.py
#!/usr/bin/python

import sys

class swap_case:
 	
 	line_in_file = ""
	with open(sys.argv[1], 'r') as test_cases:
		for line in test_cases:

			word = ""
			line_in_file = line.strip().split(' ')
			for pos in line_in_file:
				
				for i in range(len(pos)):

					if pos[i].islower():
						word += pos[i].upper()

					elif pos[i].isupper():
						word += pos[i].lower()

					else:
						word += pos[i]

				word += " "	

			print word[:-1]

			#print line_in_file


x = swap_case()

File: challenges/codeeval/easy/156/Skhorn.js
var fs  = require("fs");
var array = [];
var new_line = "";
var upper = true;
var lower = true;
fs.readFileSync(process.argv[2]).toString().split('\n').forEach(function (line) {
    if (line !== "") {
        //array = line.split(/[\s,'.:]+/)
        //console.log(array);

        for(var i = 0; i < line.length; i++)
        {
            if(line[i] == ' ' || line[i] == ',' || line[i] == ':' || line[i] == '.')
            {
                new_line += line[i];
                continue;
            }
            else
            {
                if(upper)
                {   
                    new_line += line[i].toUpperCase();
                    //console.log(new_line);
                    upper = false;
                    lower = true;
                }
                else if(lower)
                {
                    new_line += line[i].toLowerCase();
                    //console.log(new_line);
                    upper = true;
                    lower = false;
                }
            }
        }
        console.log(new_line);
        new_line = "";
        upper = true;
        lower = true;
    }
});